### PR TITLE
[gnupg] Adds sqlite as gnupg dependency

### PR DESCRIPTION
**Pull request by catenacyber**
Should fix https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=19806

### CODE REVIEW

**Comment by catenacyber** [comment]
Hmm.. we need sqlite 3.14 and package is only version 3.11

What is the best way to fix this kind of problem ?

**Comment by jonathanmetzman** [comment]
> Hmm.. we need sqlite 3.14 and package is only version 3.11
> 
> What is the best way to fix this kind of problem ?

Just download and build it.
SQLite is legendarily easy to build because you can get the source code as one file and compile it with a one liner.

File: projects/gnupg/Dockerfile [hunk 1]
```diff
@@ -16,8 +16,9 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER p.antoine@catenacyber.fr
-RUN apt-get update && apt-get install -y make autoconf automake libtool gettext bzip2 gnupg bison flex
+RUN apt-get update && apt-get install -y make autoconf automake libtool gettext bzip2 gnupg bison flex libsqlite3-dev pkg-config
```
Comment (Dor1s) [inline projects/gnupg/Dockerfile]: do you still need `libsqlite3-dev` here?

Comment (catenacyber) [reply projects/gnupg/Dockerfile]: I guess not
But I would rather like this than the manual compiling, don't you think ?

Comment (Dor1s) [reply projects/gnupg/Dockerfile]: aren't lines 20-25 in the build.sh manually compiling and installing a particular version on top of this one?

Comment (catenacyber) [reply projects/gnupg/Dockerfile]: Yes

**Comment by catenacyber** [comment]
Travis is indeed unhappy.
That seems to be another problem as the sqlite one... I will investigate later

**Comment by catenacyber** [comment]
Upstream repo seems to have fixed sqlite issue, but now we needed some change for the fuzz targets

**Comment by Dor1s** [comment]
Sorry, what's the status here? Do you need any help from us?

**Comment by catenacyber** [comment]
Max, this looks good now except for the license in the fuzz target.
Should I add them ?

**Comment by inferno-chromium** [comment]
> Max, this looks good now except for the license in the fuzz target.
> Should I add them ?

Yes please add, so travis can verify end-to-end and follows repo guildelines.

**Pull request merged**